services/core-web/src/components/modalContent/AddBondModal.js

```diff
@@ -14,20 +15,24 @@ const propTypes = {
   title: PropTypes.string.isRequired,
   bondTypeOptions: PropTypes.arrayOf(CustomPropTypes.dropdownListItem).isRequired,
   provinceOptions: PropTypes.arrayOf(CustomPropTypes.dropdownListItem).isRequired,
+  bondDocumentTypeDropDownOptions: PropTypes.arrayOf(CustomPropTypes.dropdownListItem).isRequired,
   permitGuid: PropTypes.string.isRequired,
-  bond: CustomPropTypes.bond.isRequired,
+  mineGuid: PropTypes.string.isRequired,
+  bond: CustomPropTypes.bond,
   editBond: PropTypes.bool,
 };
 
 const defaultProps = {
+  bond: {},
   editBond: false,
 };
 
 export const AddBondModal = (props) => {
-  const handleAddBond = (values) =>
-    props.editBond
+  const handleAddBond = (values) => {
+    return props.editBond
```

Comment:
I prefer the implicit return and we have done that in most places. Is there a reason you added return?

Reply:
Hmm... Can't remember. I put it back to normal.